client.go

```diff
@@ -142,24 +147,31 @@ func newClient(zkquorum string, options ...Option) *client {
 		regionReadTimeout:   region.DefaultReadTimeout,
 		done:                make(chan struct{}),
 		newRegionClientFn:   region.NewClient,
+		logger:              slog.Default(),
 	}
 	for _, option := range options {
 		option(c)
 	}
+	c.logger.Info("Creating new admin client.", "Host", slog.StringValue(zkquorum))
```

Comment:
I think this log message is wrong, it's not an admin client no?

Reply:
Thanks for catching. Fixed: https://github.com/tsuna/gohbase/pull/257/files#diff-4b667feae66c9d46b21b9ecc19e8958cf4472d162ce0a47ac3e8386af8bbd8cfR146